.gitignore

```diff
@@ -113,6 +113,7 @@ local.properties
 /server/test/gen
 /server/parser/gen
```

Comment:
probably /server/*/gen can be removed

Reply:
Parser's gen folder still exists for me.

Reply:
Removed the listings that do not have a gen folder on my local machine.